src/appkit.cs

```diff
@@ -21600,6 +21728,18 @@ interface NSWorkspace : NSWorkspaceAccessibilityExtensions {
 		[Mac (10,14)]
 		[Export ("requestAuthorizationOfType:completionHandler:")]
 		void RequestAuthorization (NSWorkspaceAuthorizationType type, Action<NSWorkspaceAuthorization, NSError> completionHandler);
+
+		[Mac (10,15)]
+		[Export ("openApplicationAtURL:configuration:completionHandler:")]
+		void OpenApplication (NSUrl applicationURL, NSWorkspaceOpenConfiguration configuration, [NullAllowed] Action<NSRunningApplication, NSError> completionHandler);
+
+		[Mac (10,15)]
+		[Export ("openURL:configuration:completionHandler:")]
+		void OpenURL (NSUrl url, NSWorkspaceOpenConfiguration configuration, [NullAllowed] Action<NSRunningApplication, NSError> completionHandler);
```

Comment:
```suggestion
		void OpenUrl (NSUrl url, NSWorkspaceOpenConfiguration configuration, [NullAllowed] Action<NSRunningApplication, NSError> completionHandler);
```

3 letter, PascalCase